packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/TransactionsPendingRow.tsx

```diff
@@ -163,24 +163,31 @@ const TransactionsCryptoRow: React.FC<TProps> = ({ transaction }) => {
                     </div>
                 )}
             </div>
-            <button
-                className='wallets-transactions-pending-row__transaction-status'
-                onClick={onMobileStatusClick}
-                ref={statusRef}
-            >
-                <Tooltip alignment='left' isVisible={!isMobile && isStatusHovered} message={transaction.description}>
-                    <div
-                        className={classNames(
-                            'wallets-transactions-pending-row__transaction-status-dot',
-                            `wallets-transactions-pending-row__transaction-status-dot--${transaction.status_code
-                                .toLowerCase()
-                                .replace('_', '-')}`
-                        )}
-                    />
-                </Tooltip>
-                <WalletText color='general' size='sm'>
-                    {transaction.status_name}
-                </WalletText>
+            <div className='wallets-transactions-pending-row__transaction-status'>
+                <button
+                    className='wallets-transactions-pending-row__transaction-status-button'
+                    data-testid='dt_transaction_status_button'
+                    onClick={onMobileStatusClick}
+                    ref={statusRef}
+                >
+                    <Tooltip
+                        alignment='left'
+                        isVisible={!isMobile && isStatusHovered}
+                        message={transaction.description}
+                    >
+                        <div
+                            className={classNames(
+                                'wallets-transactions-pending-row__transaction-status-dot',
+                                `wallets-transactions-pending-row__transaction-status-dot--${transaction.status_code
+                                    .toLowerCase()
+                                    .replace('_', '-')}`
+                            )}
+                        />
+                    </Tooltip>
+                    <WalletText color='general' size='sm'>
+                        {transaction.status_name}
+                    </WalletText>
+                </button>
```

Comment:
To resolve error `validateDOMNesting(...): <button> cannot appear as a descendant of <button>`